src/libstd/net/ip.rs

```diff
@@ -59,6 +59,48 @@ pub enum Ipv6MulticastScope {
     Global
 }
 
+impl IpAddr {
+    /// Returns true for the special 'unspecified' address (0.0.0.0 in IPv4, :: in IPv6).
+    pub fn is_unspecified(&self) -> bool {
```

Comment:
@sfackler Should I use `feature="ip"` and `issue="27709"` or should I use a different feature or issue? (#27709 and `feature="ip"` are what the Ipv4/v6Addr convenience methods already use.)